cmake 3.x policy CMP0002 forbids same target name to be defined several times.
But the 28 po/*/CMakeList.txt files do it, so allow it or patch each.

Error message was:
CMake Error at /usr/share/apps/cmake/modules/FindGettext.cmake:232 (ADD_CUSTOM_TARGET):
  add_custom_target cannot create target "pofiles" because another target
  with the same name already exists.  The existing target is a custom target
  created in source directory "/builddir/kdevplatform-1.7.3/po/fi".  See
  documentation for policy CMP0002 for more details.
Call Stack (most recent call first):
  po/gl/CMakeLists.txt:2 (GETTEXT_PROCESS_PO_FILES)

--- CMakeLists.txt.orig	2016-01-22 19:05:47.000000000 +0100
+++ CMakeLists.txt	2016-04-15 01:18:05.456531820 +0200
@@ -140,4 +140,6 @@
 configure_file(${CMAKE_SOURCE_DIR}/CTestCustom.cmake ${CMAKE_BINARY_DIR}/CTestCustom.cmake)
 
 include(MacroOptionalAddSubdirectory)
+set_property(GLOBAL
+             PROPERTY ALLOW_DUPLICATE_CUSTOM_TARGETS 1)
 macro_optional_add_subdirectory( po )
